content/posts/lets-code-contact-form-in-phoenix-part-one.md

```diff
@@ -5,15 +5,15 @@ comments: false
+In this _Let's Code_ series, I will be walking through how I added a simple contact form to a [Phoenix](https://www.phoenixframework.org/) 1.4 web application. I will try to make this as general purpose as possible so that it's easier to follow _and_ apply to your Phoenix applications. I encourage you to follow along, either in an existing Phoenix application or [a new one](https://hexdocs.pm/phoenix/up_and_running.html#content).
 
-The contact form will take the minimal amount of fields from the user (an email, subject, and a body) and send that information to a support email address. In this post, I will be walking through the schema used to model the message from the user. Let's get started.
+The contact form will take the minimal number of fields from the user (email, subject, and body) and send that information to a support email address. In this post, I will be walking through the schema used to model the message from the user. Let's get started.
 
 ## Starting with a Type Specification
 
-I like to start most new features with a [type specification](https://hexdocs.pm/elixir/typespecs.html#user-defined-types). This helps me organization and visualize the data structure(s) that I think I'll need to build the feature. Once defined, I can take this type specification and turn it into [an Elixir struct](https://hexdocs.pm/elixir/Kernel.html#defstruct/1) or [an Ecto schema](https://hexdocs.pm/ecto/Ecto.Schema.html).
+I like to start most new features with a [type specification](https://hexdocs.pm/elixir/typespecs.html#user-defined-types). This helps me organize and visualize the data structure(s) that I think I'll need to build the feature. Once defined, I can take this type specification and turn it into [an Elixir struct](https://hexdocs.pm/elixir/Kernel.html#defstruct/1) or [an Ecto schema](https://hexdocs.pm/ecto/Ecto.Schema.html).
 
-In the case of the contact form, a _message_ from the user to the support team sounds like the only type specification I will need. I will begin with a message module nestled under a `Support` context. If you're unfamiliar with the concept of contexts within Phoenix, I encourage you to read more about them [in the Phoenix guides](https://hexdocs.pm/phoenix/contexts.html#content).
+In the case of the contact form, a _message_ from the user to the support team sounds like the only type specification I will need. I begin with a `Message` module nestled under a `Support` context. If you're unfamiliar with the concept of contexts within Phoenix, I encourage you to read more about them [in the Phoenix guides](https://hexdocs.pm/phoenix/contexts.html#content). For now, you can think of the `Support` context as a boundary and public interface to all support-related functionality for the application.
```

Comment:
I like the additional sentence. I think when I see `support` my brain tends to go to `spec/support` and this helps clarify this is support as in support team and not a special, reserved term. 

Potentially, you can re-iterate "support team" again, but I don't think that it's necessary. 

```suggestion
In the case of the contact form, a _message_ from the user to the support team sounds like the only type specification I will need. I begin with a `Message` module nestled under a `Support` context. If you're unfamiliar with the concept of contexts within Phoenix, I encourage you to read more about them [in the Phoenix guides](https://hexdocs.pm/phoenix/contexts.html#content). For now, you can think of the `Support` context as a boundary and public interface to all support team-related functionality for the application.
```